### PR TITLE
docs: swift guide for connect

### PR DESCRIPTION
**Pull request by Hallidayo**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Supabase Connect - Swift

## What is the current behavior?

No current Swift example on the connect examples

## What is the new behavior?

Swift example added to connect:

https://github.com/supabase/supabase/assets/22655069/805d03de-877d-4897-8e8d-056964ca403d

## Additional context

@grdsdev  - Is it okay if you check this over and is it okay for a basic Swift example?

### CODE REVIEW

File: apps/studio/components/interfaces/Home/Connect/content/swift/supabaseswift/content.tsx [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  ConnectTabContent,
+} from 'components/interfaces/Home/Connect/ConnectTabs'
+import SimpleCodeBlock from 'components/to-be-cleaned/SimpleCodeBlock'
+
+const ContentFile = ({ projectKeys }: ContentFileProps) => {
+  return (
+    <ConnectTabs>
+      <ConnectTabTriggers>
+        <ConnectTabTrigger value="Supabase.swift" />
+        <ConnectTabTrigger value="ToDoModel.swift" />
```
Comment (grdsdev) [inline apps/studio/components/interfaces/Home/Connect/content/swift/supabaseswift/content.tsx:16]:
```suggestion
        <ConnectTabTrigger value="Todo.swift" />
```

File: apps/studio/components/interfaces/Home/Connect/content/swift/supabaseswift/content.tsx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+let supabase = SupabaseClient(
+  supabaseURL: URL(string: "${projectKeys.apiUrl ?? 'your-project-url'}")!,
+  supabaseKey: "${projectKeys.anonKey ?? 'your-anon-key'}"
+)
+        `}
+        </SimpleCodeBlock>
+      </ConnectTabContent>
+
+      <ConnectTabContent value="ToDoModel.swift">
```
Comment (grdsdev) [inline apps/studio/components/interfaces/Home/Connect/content/swift/supabaseswift/content.tsx:34]:
```suggestion
      <ConnectTabContent value="Todo.swift">
```

**Review by grdsdev — CHANGES_REQUESTED** [review]
Thanks for adding this, just requested some examples simplification.

**Comment by Hallidayo** [comment]
@grdsdev - Thank you for checking over, I've just committed the suggestions.

**Review by grdsdev — APPROVED** [review]

**Review by ivasilov — APPROVED** [review]
Tested on Studio staging, looks good.

**Pull request merged**